src/main/java/com/hubspot/jinjava/lib/filter/RenderFilter.java

```diff
@@ -38,4 +46,20 @@ public Object filter(Object var, JinjavaInterpreter interpreter, String... args)
     }
     return interpreter.render(Objects.toString(var));
   }
+
+  private void removeModuleErrors(JinjavaInterpreter jinjavaInterpreter) {
+    Stack<TemplateError> validErrors = new Stack<>();
+    while (jinjavaInterpreter.getLastError().isPresent()) {
+      TemplateError te = jinjavaInterpreter.getLastError().get();
+      jinjavaInterpreter.removeLastError();
+      if (te.getReason() == ErrorReason.DISABLED && te.getMessage().contains("module")) {
+        continue;
+      }
+      validErrors.add(te);
+    }
+
+    while (!validErrors.empty()) {
+      jinjavaInterpreter.addError(validErrors.pop());
+    }
```

Comment:
I imagine this could be quite slow, especially since in most cases there won't be any errors. We could count the number of errors present before calling `interpreter.render` and only check those which are new errors